ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -996,12 +1017,74 @@ let rank_hosts_by_best_vgpu ~__context vgpu visible_hosts =
                0L
         )
         hosts
-      |> List.map (fun g -> List.map (fun (h, _) -> h) g)
+      |> List.map (fun g -> List.map fst g)
+
+(* Group all hosts to 2 parts:
+   1. Home server. A list of home server (only one host).
```

Comment:
In xapi, it can be called "host affinity" which complies with the name of "VM.affinity" field.

Reply:
How about `affinity host`?